recipes/mikado/conda_build_config.yaml

```diff
@@ -0,0 +1,7 @@
+python:
+  - 3.6
+  - 3.7
+  - 3.8
+numpy:
+  - 1.17
+  - 1.18
```

Comment:
Can we get rid of this? We'll start building for python 3.8 soon and numpy can be restricted in the meta.yaml.

Reply:
Yes, I introduced the file when trying to understand the reason of the test failures. I can remove it now.